src/parser.c

```diff
@@ -4794,6 +4889,28 @@ void read_func_body(func_t *func)
     basic_block_t *body = read_code_block(func, NULL, NULL, func->bbs);
     if (body)
         bb_connect(body, func->exit, NEXT);
+
+    for (int i = 0; i < backpatch_bb_idx; i++) {
+        basic_block_t *bb = backpatch_bb[i];
+        insn_t *g = bb->insn_list.tail;
+        label_t *label = find_label(g->str);
+        if (!label)
+            error("goto label undefined");
```

Comment:
We should add a continue statement when a label is not found. Otherwise, this will cause a potential NULL pointer dereference.